environs/config/config.go

```diff
@@ -516,18 +516,15 @@ func ConfigDefaults() map[string]interface{} {
 	return defaultConfigValues
 }
 
-func (c *Config) ensureUnitLogging() error {
+func (c *Config) setLoggingFromEnviron() error {
 	loggingConfig := c.asString("logging-config")
 	// If the logging config hasn't been set, then look for the os environment
 	// variable, and failing that, get the config from loggo itself.
 	if loggingConfig == "" {
 		if environmentValue := os.Getenv(osenv.JujuLoggingConfigEnvKey); environmentValue != "" {
-			loggingConfig = environmentValue
-		} else {
-			loggingConfig = loggo.LoggerInfo()
+			c.defined["logging-config"] = loggingConfig
```

Comment:
s/loggingConfig/environmentValue

Reply:
Seems like there should be a failing test for this typo, it not maybe we need one to check that config from environ works

Reply:
Ah, so in the TestLoggingConfigFromEnvironment test, we do

s.PatchEnvironment(osenv.JujuLoggingConfigEnvKey, "<root>=INFO")

this is now the default value so we need to fix the test to patch something else.